apis/metacontroller/v1alpha1/types_generic.go

```diff
@@ -290,9 +296,9 @@ type GenericControllerList struct {
 	Items           []GenericController `json:"items"`
 }
 
-// Key formats the GenericController value into a
-// suitable key format
-func (gc GenericController) Key() string {
+// AsNamespaceNameKey formats the GenericController value into a
+// suitable key format that makes use of namespace & name
+func (gc GenericController) AsNamespaceNameKey() string {
```

Comment:
why not uid as a key?

Reply:
I mean can we use uid as a key?

Reply:
this is used as controller name & hence will be easy to understand than a key.